docs/zkapps/tutorials/anonymous-message-board.mdx

```diff
@@ -2,187 +2,198 @@
-  `npm run coverage`, etc.
-- A Git repo will be initialized in the project directory automatically. For
-  consistency, we use `main` as the default Git branch, by convention.
-- A [Github Actions CI workflow](https://github.com/o1-labs/zkapp-cli/tree/main/templates/project-ts/.github/workflows/ci.yml) is
-  also included. If you push your project to Github, Github Actions will run
-  your tests (named as `*.test.js`) automatically, whenever you push a commit or
-  open a pull request.
-- Code style consistency (via Prettier) and linting (via ES Lint) are
-  automatically enforced using Git pre-commit hooks. This requires no
-  configuration and occurs automatically when you commit to Git--e.g. `git commit -m 'feat: add awesome feature'`.
-- To skip all checks in the Git pre-commit hook (not recommended), you can pass
-  the `-n` flag to Git--e.g. `git commit -m 'a bad commit' -n`. But we'd
-  recommend avoiding this and resolving any errors which exist in your project
-  until the pre-commit hook passes.
+1. Create or change to a directory where you have write privileges.
+1. Create the project:
 
-### Scaffolding
+  ```sh
+  $ zk project zk file message
+  ```
 
-Now that your project is set up, you can open it in your IDE or `cd message-board` if you work from the command line.
+  The `zk project` command has the ability to scaffold the UI for your project. For this tutorial, select `none`:
 
-There should be an example smart contract in `./src` called `Add.ts` and tests for it in `Add.test.ts`. These are just example code, and you can delete them if they bother you.
+  ```
+  ? Create an accompanying UI project too? …
+    next
+    svelte
+    nuxt
+    empty
+  ❯ none
+  ```
+  
+  The `zk project` command creates a directory with a new project template that is fully set up and ready for local development. Like all zk projects, a git repository is initialized in the project directory. By convention, the `main` branch is the default branch. 
 
-Let’s create a new contract by running:
+1. Change to the project directory:
 
-```sh
-$ zk file message
-```
+  ```sh 
+  cd message-board
+  ```
+
+  For this tutorial, you run commands from the root of the `message-board` directory as you work in the `src` directory on files that contain the TypeScript code for the smart contract. 
+  
+  Each time you make updates, then build or deploy, the TypeScript code is compiled into JavaScript in the `build` directory. See the included [README](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/anonymous-message-board/README.md) file with usage instructions.
+
+### Prepare the project
+
+Example smart contract files come with the new project, you can delete them if you want.  
+
+1. Optional: Delete the example files not used in this tutorial:
+
+  ```sh
+  $ rm src/Add.ts
+  $ rm src/Add.test.ts
+  ```
+
+1. To create the `src/message.ts` file and generate the corresponding `src/message.test.ts` test file:
```

Comment:
```suggestion
1. To create the `src/message.ts` file:
```

Reply:
`zk file message` will generate a file `message.ts`, but it does not create a test file `message.test.ts`.